blogposts/announcing-sourcegraph-3.3.md

```diff
@@ -1,44 +1,217 @@
 ---
-title: 'Sourcegraph 3.3: A great title will go here'
+title: 'Sourcegraph 3.3: Improving the user experience for site admins'
 author: Ryan Blunden
-publishDate: 2019-04-20T05:59-06:00
+publishDate: 2019-04-22T10:00-07:00
 tags: [
   blog
 ]
-slug: sourcegraph-3.3
-heroImage: /blog/sourcegraph-mark
-published: false
+slug: sourcegraph-3
+heroImage: /sourcegraph-mark.png
+published: true
 ---
 
-<!--
-  Don't forget to check this against the ../STYLEGUIDE.md 
--->
-
 *Sourcegraph is an open-source, self-hosted, cross-repository code search and navigation tool, with an efficient web interface and "feels-like-native" integration into your code host. [Install or upgrade Sourcegraph](#install-or-upgrade).*
 
-INTRODUCTION
+---
 
-- [Link to each section](#section-link)
-- [Changelog](#changelog)
+<h3 style="text-align: center">Deploy or upgrade Sourcegraph 3.3</h3>
 
-## SECTION
+<div style="text-align: center">
 
-<!-- 
-Each section should have a screencast when possible, usually without sound as they only for 30s max.
+  [Local](https://docs.sourcegraph.com/#quickstart-guide) | [AWS](https://github.com/sourcegraph/deploy-sourcegraph-aws) | [DigitalOcean](https://marketplace.digitalocean.com/apps/sourcegraph?action=deploy&refcode=48dfb3ccb51c) | [Kubernetes](https://github.com/sourcegraph/deploy-sourcegraph)
 
-  <p class="container">
-    <div style="padding:56.25% 0 0 0;position:relative;">
-      <iframe src="https://player.vimeo.com/video/$VIMEO_VIDEO_ID?color=0CB6F4&amp;title=0&amp;byline=0&amp;portrait=0&autoplay=1&loop=1" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
-    </div>
-    <p style="text-align: center"><a href="https://vimeo.com/VIMEO_VIDEO_ID" target="_blank">View on Vimeo</a></p>
-  </p>
--->
+Want to try pre-release development builds? Change the Docker image tag to  `sourcegraph:insiders`.
+
+</div>
+
+---
+
+This release focusses on making Sourcegraph easier to set up, configure and manage repositories.
+
+We’ve completely overhauled how repositories are selected and kept in-sync, benefiting everyone, but especially organizations with thousands (or tens of thousands) of repositories. We’ve also added Sentry to our list of third party developer tool integrations and improved Swift code intelligence and navigation.
```

Comment:
```suggestion
We’ve completely overhauled how repositories are mirrored from your code host so that it is easier to configure and more reliable. This benefits everyone, but especially organizations with thousands (or tens of thousands) of repositories. We’ve also added Sentry to our list of third party developer tool integrations and improved Swift code intelligence and navigation.
```